proofzero/rollupid
# feat(console): Group seats assignment

### Description

<img width="1446" alt="image" src="https://github.com/proofzero/rollupid/assets/635806/1687c983-baf1-4828-a90f-046f04569b4f">

### Related Issues

- Closes #2674 

### Testing

- Filled free group members, notice that add member is disabled
- Manually remove disabled, try to add member, notice 500 error
- Remove free-member limit and introduce paid seat extra limit
- Fill free + paid members, notice that they get colored side-bars, notice that add member is disabled
- Manually remove disabled, try to add member, notice 500 error

### Checklist
